go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	github.com/gorilla/mux v1.8.1
 	github.com/in-toto/attestation v1.1.0
 	github.com/sigstore/cosign/v2 v2.2.4
-	github.com/sigstore/sigstore-go v0.2.0
+	github.com/sigstore/sigstore-go v0.4.0
```

Comment:
You can bump to 0.5.1 now, which does have a few breaking changes in API calls, but I don't think you're affected.

Reply:
added